src/data/common/Dhis2DataElement.ts

```diff
@@ -83,7 +87,11 @@ function makeCocOrderArray(namesArray: string[][]): string[] {
 function getCocOrdered(categoryCombo: D2CategoryCombo, config: Dhis2DataStoreDataForm) {
     const allCategoryOptions = categoryCombo.categories
         .map(c => {
-            return c.categoryOptions.flatMap(co => ({ name: co.name, shortName: co.shortName, formName: co.formName }));
+            return c.categoryOptions.flatMap(co => ({
+                name: co.name,
+                shortName: co.shortName,
+                formName: co.displayFormName,
```

Comment:
should we use also the translated fields for name and shortName?

Reply:
we already have customizations in the autogen-config to display either formName or shortName (for the grid-with-combos) viewtype